authentik/core/models.py

```diff
@@ -41,19 +42,27 @@
 USER_ATTRIBUTE_DELETE_ON_LOGOUT = "goauthentik.io/user/delete-on-logout"
 USER_ATTRIBUTE_SOURCES = "goauthentik.io/user/sources"
 USER_ATTRIBUTE_TOKEN_EXPIRING = "goauthentik.io/user/token-expires"  # nosec
+USER_ATTRIBUTE_TOKEN_MAXIMUM_LIFETIME = "goauthentik.io/user/token-maximum-lifetime"  # nosec
 USER_ATTRIBUTE_CHANGE_USERNAME = "goauthentik.io/user/can-change-username"
 USER_ATTRIBUTE_CHANGE_NAME = "goauthentik.io/user/can-change-name"
 USER_ATTRIBUTE_CHANGE_EMAIL = "goauthentik.io/user/can-change-email"
 USER_PATH_SYSTEM_PREFIX = "goauthentik.io"
 USER_PATH_SERVICE_ACCOUNT = USER_PATH_SYSTEM_PREFIX + "/service-accounts"
 
+DEFAULT_TOKEN_DURATION = "minutes=30"  # nosec
```

Comment:
I wonder if we should make this configurable as we do with the token length.

Reply:
we already have `default_token_length` so we can also add `default_token_duration` as config (we'll also have to validate the duration on config load)

Reply:
I added this, w/o the validation for now. How do you want to handle this?

Reply:
Once this is in, I'll also open a follow-up PR to migrate those options to be per-tenant.

Reply:
Actually I added this directly in here.